users/views.py

```diff
@@ -54,12 +54,14 @@ def post(self, request):
         data          = json.loads(request.body)
 
         try:
+            if data['signup_type'] != 'flix':
+                return JsonResponse({'message': 'LOGIN_TYPE_IS_NOT_FLIX'}, status=400)
```

Comment:
`flix`를 하드 코딩하는 방법 말고 Enum class를 사용해서 구현해주세요.

```python
from enum import Enum

class SignUpType(Enum):
    FLIX = 1
    KAKAO = 2
```

```suggestion
            if data['signup_type'] != SignUpType.FLIX.name
                return JsonResponse({'message': 'LOGIN_TYPE_IS_NOT_FLIX'}, status=400)
```

Reply:
@lordmyshepherd 
이렇게 enum 으로 변경하는 이유는
데이터베이스에 가벼운 int 값을 넣어서 가볍게 관리하고자 하는 목적,
혹은 통신 자체를 가볍게 하기 위한 목적 
두 가지 정도로 생각할 수 있었는데, 병민님은 어떤 걸 의도하신 건지 궁금합니다. 혹은 완전히 다른 이유였을까요?
모델 자체를 변경해야하나 고민이 되어서 질문드립니다.